dev/process.py

```diff
@@ -0,0 +1,24 @@
+#! /opt/homebrew/bin/python3
```

Comment:
should this shebang be here? its in `/usr/local/bin/python3` on my machine

Reply:
Probably not, I was using it for debugging :)